velodrome/fetcher/conversion.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/test-infra/ciongke/github"
+)
+
+// NewIssue creates a new (orm) Issue from a github Issue
+func NewIssue(gIssue *github.Issue) (*Issue, error) {
+	if gIssue.Number == nil ||
```

Comment:
`ciongke`'s `github` package doesn't use pointers so these will be their 0 value, not nil. Does this even compile?